script/build_hosts

```diff
@@ -80,9 +84,9 @@ judges:
         print_server("balloonmanager", "10.4.4.149", others)
         # print_server("kattis", "10.3.3.212", others)
         # print_server("kattisbackup", "10.3.3.213", others)
-        # print_server("pc2-shadowccs", "10.3.3.214", others)
+        print_server("pc2-shadowccs", "10.3.3.214", others)
         # print_server("domjudge-shadowccs", "10.3.3.215", others)
-        # print_server("domserver", "10.3.3.216", others)
+        print_server("domserver", "10.3.3.216", others)
```

Comment:
do we need a `domserver-backup` or similar?

Reply:
will do